src/modules/Invoice/Service.php

```diff
@@ -1513,6 +1472,17 @@ private function getPdfCss(): string
         return $CSS;
     }
 
+    private function getPdfTemplate(): string
+    {
+        $basePath = __DIR__ . DIRECTORY_SEPARATOR . 'pdf_template' . DIRECTORY_SEPARATOR;
```

Comment:
Since the path is only used once, we don't need to define a new var for it and instead can just write it out on line 1479